playbooks/common/openshift-master/config.yml

```diff
@@ -228,7 +228,7 @@
   gather_facts: no
   tasks:
   - set_fact:
-      haproxy_backend_servers: "{{ hostvars | oo_select_keys(groups['oo_masters_to_config']) | oo_haproxy_backend_masters }}"
+      haproxy_backend_servers: "{{ hostvars | oo_select_keys(groups['oo_masters']) | oo_haproxy_backend_masters }}"
```

Comment:
Instead of set_fact here, this should probably be set as a var for the role directly..
```
roles:
- role: haproxy
  haproxy_backend_servers: "{{ hostvars | oo_select_keys(groups['oo_masters']) | oo_haproxy_backend_masters }}"
```

We might want to add some docs somewhere to say not to run ansible against the lb group without also specifying the masters group